test/transport_web/solutions/login_test.exs

```diff
@@ -39,4 +41,16 @@ defmodule TransportWeb.LoginTest do
     # I can click somewhere to ask for help
     assert visible_page_text() =~ "Nous contacter"
   end
+
+  @tag :integration
```

Comment:
I don't think this test is needed, I'd personally save "solution" tests for user journeys.